crates/swc_ecma_transforms_compat/tests/es2015_classes.rs

```diff
@@ -6347,7 +6347,7 @@ test!(
 
         chain!(
             es2022::es2022(es2022::Config { loose: false }),
-            es2018::es2018(),
+            es2018::es2018(Default::default()),
```

Comment:
Is change is for future changes? I'm fine with it anyway, but just to be sure.

Reply:
Yes, in case `swc` introduces assumptions like babel, which has 4 assumptions for `object_spread`

Reply:
Great, thanks!